app/Livewire/Project/Database/BackupExecutions.php

```
@@ -110,6 +124,81 @@ public function download_file($exeuctionId)
         return redirect()->route('download.backup', $exeuctionId);
     }
 
+    public function confirmRestore(int $executionId)
+    {
+        $execution = ScheduledDatabaseBackupExecution::find($executionId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\ScheduledDatabaseBackupExecution' in method 'confirmRestore'. (undefined)

(StaticAccess)